src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -4,12 +4,14 @@
 #include <QList>
 #include <QLocale>
 #include <QScreen>
+#include <QVariant>
```

Comment:
not required

Reply:
I used `QVariant::fromValue` further below, so I felt it was appropriate to include as per "include what you use".

Reply:
sure, but the DlgPef.. dialogs are QWidgets which already bring QVariant IINM

Reply:
yeah I understand that reasoning but that would mean that we'd rely on transitive dependencies, which is bad practice. In reality it doesn't make any difference, but I think leaving it there is better practice than removing (which is why I added it in the first place).